timetagger_cli/core.py

```diff
@@ -154,6 +154,70 @@ def stop():
         print_records(running_records)
 
 
+def resume():
+    """Start a timer with the same settings as previous"""
+
+    now = int(time.time())
+    d = datetime.datetime.fromtimestamp(now)
+
+    today = datetime.datetime(d.year, d.month, d.day)
+    tomorrow = today + datetime.timedelta(1)
+
+    t1 = int(today.timestamp())
+    t2 = int(tomorrow.timestamp())
+
+    # Get records from today
+    records = request("GET", f"records?timerange={t1}-{t2}")["records"]
+
+    if len(records) == 0:
+        print("No records earlier today.")
+        return 
+
+    # Remove HIDDEN records
+    filtered_records = []
+    for r in records:
+        if "HIDDEN" in r["ds"]:
+            continue
+
+        filtered_records.append(r)
```

Comment:
Great place for a list comprehension:
```suggestion
    filtered_records = [r for r in records if "HIDDEN" not in r["ds"]]
```